app/templates/components/speaker-item.hbs

```diff
@@ -0,0 +1,27 @@
+<div class="speaker-item">
+  {{#if speaker.image}}
+    <img alt="" class="ui small picture" src="{{speaker.image}}">
+  {{else}}
+    <img alt="" class="ui small picture" src="/images/placeholders/avatar.png">
+  {{/if}}
+  <br>
+  <div>{{ speaker.name }}</div>
+  <div>{{ speaker.organisation }}</div>
+  <div class="social-links">
+    <a href="#" class="home-button">
+      <i class="home icon"></i>
+    </a>
+    <a href="#" class="facebook-button">
+      <i class="facebook f icon"></i>
+    </a>
+    <a href="#" class="twitter-button">
+      <i class="twitter icon"></i>
+    </a>
+    <a href="#" class="linkedin-button">
```

Comment:
Instead of a link with a background color, use semantic-ui's square icons. --> https://semantic-ui.com/elements/icon.html#bordered 

Because, if you use a link with background color, there is a slight mis-alignment. 

<img width="129" alt="screen shot 2017-05-08 at 10 14 43 pm" src="https://cloud.githubusercontent.com/assets/2404372/25814606/c41a033e-343b-11e7-9825-19db4bbe5451.png">